src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -264,6 +267,9 @@
                             url: url,
                             method: method,
                             type: 'fetch',
+                            updateDuration: function(element) {
```

Comment:
you need to do the same in the code handling XMLHttpRequest. This code does it only for `fetch` (which will break stuff as `startAjaxRequest` is used by both)